pkg/reconciler/kubernetes/tektonpipeline/tektonpipeline.go

```diff
@@ -379,8 +379,17 @@ func (r *Reconciler) targetNamespaceCheck(ctx context.Context, tp *v1alpha1.Tekt
 	ns, err := r.kubeClientSet.CoreV1().Namespaces().Get(ctx, tp.GetSpec().GetTargetNamespace(), metav1.GetOptions{})
 	if err != nil {
 		if apierrors.IsNotFound(err) {
+			if err := common.CreateTargetNamespace(labels, tp, r.kubeClientSet); err != nil {
+				return err
```

Comment:
from my experience an additional check on this error `isAlreadyExist` and `return nil` will keep the recociler running.

Reply:
@nikhil-thomas I guess this comment is related to PR https://github.com/tektoncd/operator/pull/600 as I have rebased over it for now